### PR TITLE
[#289] Apple will remove bitcode in Xcode 14, we should remove bitcode from our configurations

### PR DESCRIPTION
**Pull request by Shayokh144**
#289

## What happened

Apple outlined in Xcode 14 [Documentation](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes)

`Deprecations
Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.
Xcode no longer builds bitcode by default and generates a warning message if a project explicitly enables bitcode: “Building with bitcode is deprecated. Please update your project and/or target settings to disable bitcode.” The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store. Debug symbols for past bitcode submissions remain available for download. (86118779)
Currently we uses bitcode in some configurations as discussed in our document. It might be troublesome when our team adopt Xcode 14.`
 
## Insight

- write a script for searching all occurrences of bitcode in all configurations file, then change them to disable
- update build flag document
- remove bitcode for pods also
- test the work with **Xcode 14 Beta version** by creating projects by using this template

 
## Proof Of Work

**when bitcode was enabled:**
oldXcode14

**after disabling bitcode:**
newXcode14

### CODE REVIEW

File: .github/wiki/Project-Configurations.md [hunk 1]
```diff
@@ -124,15 +124,5 @@ Only include the dSYM file for release builds.
...
-Enable build with bitcode for Production only.
-
-| Build configurations | Enable Bitcode |
-|---|---|
-| Debug Staging |  |
-| Release Staging |  |
-| Debug Production |  |
-| Release Production | ✔︎ |
+### Disable Bitcode
+- Acording to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.  The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```
Comment (blyscuit) [inline .github/wiki/Project-Configurations.md:128]: Too many spacing here 😃 
```suggestion
- Acording to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14. The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```

**Review by blyscuit — APPROVED** [review]

File: .github/wiki/Project-Configurations.md (hunk 1 above)
Comment (Shayokh144) [reply .github/wiki/Project-Configurations.md]: @blyscuit update the spacing in this [commit](https://github.com/nimblehq/ios-templates/pull/318/commits/1b9b2f2f36b64626b14d2b7bce7bda3438b78d57)

Comment (suho) [reply .github/wiki/Project-Configurations.md]: @Shayokh144 @blyscuit From the release note, it said, "bitcode is no longer required for watchOS and tvOS applications". How about iOS applications?

**Review by phongvhd93 — APPROVED** [review]

File: .github/wiki/Project-Configurations.md (hunk 1 above)
Comment (Shayokh144) [reply .github/wiki/Project-Configurations.md]: in documentation is is said "Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and `the App Store no longer accepts bitcode submissions from Xcode 14.`"
If App Store no longer accepts bitcode submissions then it is good to disable that for all platforms.

File: .github/wiki/Project-Configurations.md [hunk 2]
```diff
@@ -124,15 +124,5 @@ Only include the dSYM file for release builds.
...
-Enable build with bitcode for Production only.
-
-| Build configurations | Enable Bitcode |
-|---|---|
-| Debug Staging |  |
-| Release Staging |  |
-| Debug Production |  |
-| Release Production | ✔︎ |
+### Disable Bitcode
+- Acording to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14. The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```
Comment (suho) [inline .github/wiki/Project-Configurations.md:128]: Typo with `Acording` and I think it better to add a line break 
```suggestion

According to [Xcode 14 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes): Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14. The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store.
```

**Review by suho — CHANGES_REQUESTED** [review]

File: .github/wiki/Project-Configurations.md (hunk 2 above)
Comment (Shayokh144) [reply .github/wiki/Project-Configurations.md]: @suho update spelling in this [commit](https://github.com/nimblehq/ios-templates/pull/318/commits/80379013a96a302d9b4f0b12f11f3e8ba3285cba)

**Review by suho — APPROVED** [review]

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**